go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -204,10 +197,8 @@ func (c *addressManager) runInternal(stopChan <-chan struct{}, doneWg *sync.Wait
 	klog.Info("Node IP manager is running")
 }
 
-// handleNodeChanges takes a node obj, extracts its name and determines if the node's address changed. If so, it
-// updates the node's OVS encapsulation IP.
-func (c *addressManager) handleNodeChanges(obj interface{}) {
-	var err error
+// handleNodeChanges updates OVN's EncapIP if the node IP changed
```

Comment:
```suggestion
// handleNodePrimaryAddrChange updates OVN's EncapIP if the node IP changed
```

Reply:
To avoid similar issues in the future, i.e. a comment that gets out of date because the function name changed, I've removed the function name from the comment in its entirety. The same style is used by some of the functions in this file.